clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/appgateway/cache/AzureAppGatewayCachingAgent.groovy

```diff
@@ -114,6 +114,7 @@ class AzureAppGatewayCachingAgent extends AzureCachingAgent {
 
     def cacheResult = metricsSupport.transformData {
       if (updatedAppGateway) {
+        updatedAppGateway.accountName = accountName
```

Comment:
There're couple loops in the deck/UI side where they run through each load balancer and each account in order to initialize some other structures. I've noticed that in our case we were failing because we did not provide that extra info into the description object that was retrieved from the clouddriver side.

I did not check if we run into similar issue for the other resources, server group or security group.